core/tests/unit/ldml/meson.build

```
@@ -35,20 +35,46 @@ if node.found()
   subdir('invalid-keyboards')
 endif
 
-
 # Build ldml test executable
 
+keyboard_build_path = join_paths(meson.current_build_dir(),'keyboards')
```

```suggestion
keyboard_build_path = meson.current_build_dir() / 'keyboards'
```

It's cleaner to use `/` now in meson path construction